clickhouse写入bug

**clickhouse写入bug：**
```
com.wgzhao.addax.common.exception.AddaxException: Code:[DBUtilErrorCode-05], Description:[往您配置的写入表中写入数据时失败.].  - java.sql.SQLException: Cannot rollback when auto-commit is enabled
 	at ru.yandex.clickhouse.ClickHouseConnectionImpl.rollback(ClickHouseConnectionImpl.java:230)
 	at org.apache.commons.dbcp2.DelegatingConnection.rollback(DelegatingConnection.java:482)
 	at org.apache.commons.dbcp2.DelegatingConnection.rollback(DelegatingConnection.java:482)
 	at com.wgzhao.addax.rdbms.writer.CommonRdbmsWriter$Task.doBatchInsert(CommonRdbmsWriter.java:407)
 	at com.wgzhao.addax.rdbms.writer.CommonRdbmsWriter$Task.startWriteWithConnection(CommonRdbmsWriter.java:322)
 	at com.wgzhao.addax.rdbms.writer.CommonRdbmsWriter$Task.startWrite(CommonRdbmsWriter.java:340)
 	at com.wgzhao.addax.plugin.writer.clickhousewriter.ClickHouseWriter$Task.startWrite(ClickHouseWriter.java:137)
 	at com.wgzhao.addax.core.taskgroup.runner.WriterRunner.run(WriterRunner.java:81)
 	at java.lang.Thread.run(Thread.java:748)
  - java.sql.SQLException: Cannot rollback when auto-commit is enabled
 	at ru.yandex.clickhouse.ClickHouseConnectionImpl.rollback(ClickHouseConnectionImpl.java:230)
 	at org.apache.commons.dbcp2.DelegatingConnection.rollback(DelegatingConnection.java:482)
 	at org.apache.commons.dbcp2.DelegatingConnection.rollback(DelegatingConnection.java:482)
 	at com.wgzhao.addax.rdbms.writer.CommonRdbmsWriter$Task.doBatchInsert(CommonRdbmsWriter.java:407)
 	at com.wgzhao.addax.rdbms.writer.CommonRdbmsWriter$Task.startWriteWithConnection(CommonRdbmsWriter.java:322)
 	at com.wgzhao.addax.rdbms.writer.CommonRdbmsWriter$Task.startWrite(CommonRdbmsWriter.java:340)
 	at com.wgzhao.addax.plugin.writer.clickhousewriter.ClickHouseWriter$Task.startWrite(ClickHouseWriter.java:137)
 	at com.wgzhao.addax.core.taskgroup.runner.WriterRunner.run(WriterRunner.java:81)
 	at java.lang.Thread.run(Thread.java:748)
 
 	at com.wgzhao.addax.common.exception.AddaxException.asAddaxException(AddaxException.java:66)
 	at com.wgzhao.addax.rdbms.writer.CommonRdbmsWriter$Task.startWriteWithConnection(CommonRdbmsWriter.java:327)
 	at com.wgzhao.addax.rdbms.writer.CommonRdbmsWriter$Task.startWrite(CommonRdbmsWriter.java:340)
 	at com.wgzhao.addax.plugin.writer.clickhousewriter.ClickHouseWriter$Task.startWrite(ClickHouseWriter.java:137)
 	at com.wgzhao.addax.core.taskgroup.runner.WriterRunner.run(WriterRunner.java:81)
 	at java.lang.Thread.run(Thread.java:748)
 Caused by: java.sql.SQLException: Cannot rollback when auto-commit is enabled
 	at ru.yandex.clickhouse.ClickHouseConnectionImpl.rollback(ClickHouseConnectionImpl.java:230)
 	at org.apache.commons.dbcp2.DelegatingConnection.rollback(DelegatingConnection.java:482)
 	at org.apache.commons.dbcp2.DelegatingConnection.rollback(DelegatingConnection.java:482)
 	at com.wgzhao.addax.rdbms.writer.CommonRdbmsWriter$Task.doBatchInsert(CommonRdbmsWriter.java:407)
 	at com.wgzhao.addax.rdbms.writer.CommonRdbmsWriter$Task.startWriteWithConnection(CommonRdbmsWriter.java:322)
 	... 4 common frames omitted
```

_Originally posted by @tixff in https://github.com/wgzhao/Addax/issues/491#issuecomment-1006249006_

--- a/pom.xml
+++ b/pom.xml
@@ -77,7 +77,7 @@
 
         <!-- jdbc driver version -->
         <cassandra.jdbc.version>3.11.0</cassandra.jdbc.version>
-        <clickhouse.jdbc.version>0.3.1-patch</clickhouse.jdbc.version>
+        <clickhouse.jdbc.version>0.3.0</clickhouse.jdbc.version>
         <hive.jdbc.version>3.1.2</hive.jdbc.version>
         <influxdbClient.version>2.22</influxdbClient.version>
         <hadoop.version>3.2.0-14</hadoop.version>

